.github/workflows/oci-ci.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
       - name: Output Image Metadata
         id: metadata
         run: |
-          IMAGE_NAME=$(docker compose run --rm ${{ inputs.docker-compose-service }} yq '.["image.name"]' oci-metadata.json)
+          IMAGE_NAME=$(docker compose run --rm ${{ inputs.docker-compose-service }} yq --output-format=yaml '.["image.name"]' oci-metadata.json)
```

Comment:
Shouldn't the target's extension also be `.yaml`?

Reply:
No, it is reading a file (json in this case), not writing. It writes to stdout